public/app/features/variables/editor/SelectionOptionsEditor.tsx

```diff
@@ -4,14 +4,17 @@ import { e2e } from '@grafana/e2e';
 
 import { VariableWithMultiSupport } from '../../templating/types';
 import { VariableEditorProps } from './types';
+import { VariableIdentifier, toVariableIdentifier } from '../state/types';
 
 export interface SelectionOptionsEditorProps<Model extends VariableWithMultiSupport = VariableWithMultiSupport>
-  extends VariableEditorProps<Model> {}
+  extends VariableEditorProps<Model> {
+  onMultiChanged: (identifier: VariableIdentifier, value: boolean) => void;
+}
 
 export const SelectionOptionsEditor: FunctionComponent<SelectionOptionsEditorProps> = props => {
   const onMultiChanged = useCallback(
     (event: React.ChangeEvent<HTMLInputElement>) => {
-      props.onPropChange({ propName: 'multi', propValue: event.target.checked });
+      props.onMultiChanged(toVariableIdentifier(props.variable), event.target.checked);
     },
     [props.onPropChange]
```

Comment:
This should be correct, right?
```suggestion
    [props.onMultiChanged]
```

Reply:
Great finding! That would have been a nasty bug 🙈